src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx

```diff
@@ -685,9 +685,10 @@ Prior versions of Npgsql may also be instrumented, but duplicate and/or missing
             />
           </td>
           <td>
-* Minimum supported version: 1.0.488
-* Verified compatible versions: 1.0.488, 1.1.608, 1.2.6, 2.0.601, 2.1.58, 2.2.88, 2.6.66
-		  </td>
+            * Minimum supported version: 1.0.488
+            * Verified compatible versions: 1.0.488, 1.1.608, 1.2.6, 2.0.601, 2.1.58, 2.2.88, 2.6.66
+            * Known incompatible versions: 4.0.44 and above
```

Comment:
Looks like you put this in the wrong row (you put the version in StackExchange.Redis instead of ServiceStack.Redis)

Reply:
https://github.com/newrelic/docs-website/pull/11700

Reply:
Fixed in the this PR: https://github.com/newrelic/docs-website/pull/11700